plugins/catalog-import/report.api.md

```diff
@@ -140,11 +142,13 @@ export { catalogImportPlugin };
 export { catalogImportPlugin as plugin };
 
 // Warning: (ae-forgotten-export) The symbol "StepperProvider" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "catalogImportTranslationRef" needs to be exported by the entry point index.d.ts
```

Comment:
should this be exported?

Reply:
<img width="651" alt="image" src="https://github.com/user-attachments/assets/19639f22-c192-4621-8f08-b263e73409cd" />

It's exported in the `alpha` entry like other translations.

Reply:
Friendly ping @benjdlambert

Reply:
OK, so I should add `export { catalogImportTranslationRef } from './translation';` for both two files?

Reply:
No, you just need to change the import from `import { catalogImportTranslationRef } from '../../` to `import { catalogImportTranslationRef } from '@backstage/plugin-catalog-import/alpha'`

Reply:
Done!